### PR TITLE
Make clippy happy

### PR DESCRIPTION
**Pull request by lhchavez**
Not sure why this wasn't causing problems before, but it is now.

All clippy warnings are gone.

### CODE REVIEW

File: build.rs [hunk 1]
```diff
@@ -1,5 +1,13 @@
+use std::env;
+use std::fs;
+use std::path::Path;
+
 extern crate prost_build;
 
 fn main() {
     prost_build::compile_protos(&["src/client.proto"], &["src/"]).unwrap();
+    // Make clippy happy by telling it to ignore any badness in this file.
```
Comment (cbrewster) [inline build.rs:9]: 🙃

**Review by cbrewster — APPROVED** [review]
🖇️

File: build.rs (hunk 1 above)
Comment (lhchavez) [reply build.rs]: gah, figured it out. this file is included in another, so we needed to add the ignore to the `mod` around the `include!`! https://github.com/replit/rfbproxy/pull/28/files#diff-42cb6807ad74b3e201c5a7ca98b911c5fa08380e942be6e4ac5807f8377f87fcR32

Comment (cbrewster) [reply build.rs]: oh nice!

**Pull request merged**